Signal/src/network/PushManager.m

```diff
@@ -162,6 +162,10 @@ - (void)application:(UIApplication *)application didReceiveRemoteNotification:(N
     }
 }
 
+- (void)applicationDidBecomeActive {
```

Comment:
I love it. clang-format takes a (massive) config file - are you just using defaults?